style/classic.js

```diff
@@ -0,0 +1,63 @@
+/**
+ * @license
+ * Visual Blocks Editor
+ *
+ * Copyright 2018 Google Inc.
+ * https://developers.google.com/blockly/
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * @fileoverview Classic style.
+ * Contains multi colored border to create shadow effect.
+ * @author aschmiedt@google.com (Abby Schmiedt)
```

Comment:
Since this will be public, you don't need to include your email if you don't want to. You don't even need to include any @author line, though full name or just GitHub id might be balance between privacy and credit. Technically, most of this will be in the Git logs (there is a privacy flag for that, in the preferences), but keeping it out of the file makes it harder for people to extract or find its way into other projects.